frontend/README.md

```diff
@@ -4,22 +4,26 @@ This component implements a web application that we can use to retrieve Spotify
 
 In the future, we want this to be an interface to enable and customize playlist creations for the user.
 
-# Devleopment setup
+# Development setup test
 
-Make sure Node 10.5.0+ and Make are available in your system. Then install dependencoes:
+Make sure Node 20 and Make are available in your system. Then install dependencies:
 
 ```shell
 npm install
 ```
 
+## Create new Spotify app and config the redirect URIs
+
 To run this app you need to create your own Spotify app following [these instructions](https://developer.spotify.com/documentation/web-api/tutorials/getting-started#create-an-app).
 
-Then fill the env file (i.e. `.env`) with the corresponding Spotify secrets:
+Put this URL `http://XXX:3000/api/auth/callback/spotify` as a redirect URI
```

Comment:
Maybe we cna be a bit more explicit, something like:

```
Put this URL `http://<host>:3000/api/auth/callback/spotify` as a redirect URI. We are using `localhost` as host, for now.
```

Reply:
done!